File: Python/Uri-Judge/Matematica/fazendo_nada.py
while True:
    dados = [int(x) for x in input().split()]
    if dados[0] == -1 and dados[1] == -1:
        break
    lista = [int(x) for x in input().split()]
    capitulos = 0
    tamanho = len(lista)
    for x in lista:
        capitulos += x * tamanho
        tamanho -= 1
    print(capitulos * dados[1])

File: dart/fazendo_nada.dart
import 'dart:io';

void main(List<String> args) {
  while (true) {
    var dados = stdin.readLineSync()!.split(' ').map(int.parse).toList();
    if (dados[0] == -1 && dados[1] == -1) {
      break;
    }
    var lista = stdin.readLineSync()!.split(' ').map(int.parse).toList();
    int tamanho = lista.length;
    int total = lista.fold(
      0,
      (previous, current) => previous + (current * tamanho),
    );

    print(total);
  }
}
